src/redux/login-reducer.js

```diff
@@ -0,0 +1,24 @@
+const initialState = {
+    email: '',
+    isAuth: false,
+    password: ''
+};
+
+const loginReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'SET_USER_DATA':
+            return {
+                ...state,
+                email: action.email,
+                isAuth: true,
+                password: action.password
+            };
+        default:
+            return state;
+    }
+};
+
+export const setAuthUserData = (email, password) => ({ type: 'SET_USER_DATA', email, password });
+
+
+export default loginReducer;
```

Comment:
Move it to the "store" folder. You can ask Feoktist about the approach

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/store/reducers/loginReducers.js#L1-L37